test/controllers/basic_search_controller_test.rb

```diff
@@ -6,6 +6,48 @@ class BasicSearchControllerTest < ActionDispatch::IntegrationTest
     assert_response :success
 
     assert_select 'form#basic-search', { count: 1 }
+
+    details_div = assert_select('details#advanced-search-panel')
+    assert_nil details_div.attribute('open')
+  end
+
+  test 'advanced search form appears on index with URL parameter' do
+    get '/?advanced=true'
+
+    assert_response :success
+
+    details_div = assert_select('details#advanced-search-panel')
+    refute_nil details_div.attribute('open')
+  end
+
+  test 'advanced search form appears on results page with URL parameter' do
+    VCR.use_cassette('advanced',
```

Comment:
See note in cassette. The cassette returns a 500 error but the test isn't explicitly expecting an error so I'm nervous we could be confused in the future. I'd recommend regenerating the cassette.